app/design/frontend/base/default/template/magmodules/webwinkelconnect/popup/privacy-popup.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+if (!$this->isPopupEnabled()) {
+    return;
+}
+
+echo $this->getOrderData() ? sprintf(
+    '<script type="application/json" id="%s_order_completed">%s</script>',
+    htmlentities('webwinkelkeur'),
+    json_encode($this->getOrderData(), JSON_HEX_QUOT | JSON_HEX_TAG | JSON_HEX_AMP | JSON_HEX_APOS | (JSON_INVALID_UTF8_SUBSTITUTE ?: 0))
```

Comment:
This causes a notice to be raised on PHP 7.1: https://3v4l.org/u1isdt

You should check whether the constant is [defined](https://www.php.net/defined).